### PR TITLE
Implemented Session 2

### PR DESCRIPTION
**Pull request by Akshay-jain22**

### CODE REVIEW

File: session_0/README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Rails. You can refer to books on programming or websites like
 
 > Read the [installation guide](/installation.md) and [setting up local
 > workspace](/essential_git.md) to get started.
+>
```
Comment (abhishekkumar2718) [inline session_0/README.md:20]: Please don't upload Session 0 and Session 1 related files - using branches and pushing code correctly is part of learning how to use Git.

**Review by abhishekkumar2718 — CHANGES_REQUESTED** [review]

**Comment by abhishekkumar2718** [comment]
Great job, good to see that you have been going through comments on the other PRs and learning them as well :partying_face:

File: session_0/README.md (hunk 1 above)
Comment (Akshay-jain22) [reply session_0/README.md:20]: What I did was :
git add .
git commit

So, I'm not sure why only session_1 files got added not session_0. Like, either both shd be uploaded or none of them uploaded.
Also, for upcoming sessions, should I do only 'git add session_id' to make that only session to be added in the branch.

Comment (abhishekkumar2718) [reply session_0/README.md:20]: Yes

Comment (Akshay-jain22) [reply session_0/README.md:20]: ohk..Gr8 :+1:

**Pull request closed without merging**